Assets/Scripts/Enemy/EnemyFactory.cs

```diff
@@ -51,6 +51,7 @@ public EnemyFactory()
         // Define enemyInfo for each type.
         AddTurretToRoster();
         _enemyInfo.Add(EnemyType.GRENADIER, new EnemyInfo(EnemyMovements.GrenadierMovement, Globals.DirectTargeting, EnemyMovements.GrenadierSetup, new ArtilleryAttackBehaviour(EntityType.ENEMY, 20f), 10f));
+        _enemyInfo.Add(EnemyType.RANGED, new EnemyInfo(EnemyMovements.RangedMovement, Globals.DirectTargeting, EnemyMovements.RangedSetup, new BulletAttackBehaviour(EntityType.ENEMY, 5f, 20f), 1f));
```

Comment:
Are we gonna be implementing enemies like this or do we want to put it in a method so its easier to read ?

Reply:
yes